frontend/pulse3d/src/components/table/UploadsSubTable.js

```diff
@@ -0,0 +1,71 @@
+import { useState, memo } from "react";
+import styled from "styled-components";
+
+const SubContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+const SubHeader = styled.div`
+  display: flex;
+  background-color: var(--dark-blue);
+  color: white;
+  margin: 0 1rem;
+  padding: 0.4rem;
+  justify-content: space-around;
+`;
+const SubRow = styled.div`
+  font-size: 0.75rem;
+  width: 20%;
+  text-align: center;
+  margin: 1rem 0;
+`;
+
+export default memo(function UploadsSubTable(props) {
+  const [isChecked, setIsChecked] = useState(props.jobs.map((job) => job.checked));
+  const rows = props.jobs.map((job, idx) => {
+    let paramsString = [];
+    Object.keys(job.analysisParams).forEach((param) => {
+      if (job.analysisParams[param] !== null) {
+        if (param === "peaks_valleys") {
+          paramsString.push(<div key={Math.random()}>{`${param} : user set`}</div>);
+          return;
```

Comment:
I think you could just use if/else here instead of returning, unless I'm missing something. Or better yet something like this
```
const paramVal = job.analysisParams[param] ? "user set" : job.analysisParams[param]
paramsString.push(<div key={Math.random()}>{`${param} : ${paramVal}`}</div>);
```

Reply:
I also thought it might be more user-friendly to remove the snake casing from the param names shown in this table column

Reply:
I think Tanner was saying this could be simplified

Reply:
Yeah, with the snake case change it could be this now
```
const paramVal = param === "peaks_valleys" ? "user set" : job.analysisParams[param];
paramsString.push(<div key={Math.random()}>{`${param.replace("_", " "): ${paramVal}`}</div>);
```

Also why is `Math.random()` used for the key here? If you set it to string containing the job ID + param and it should always be unique. That would also make it easier to find the element in the HTML if that ever needed to be done